### PR TITLE
Update/network switcher

### PR DESCRIPTION
**Pull request by as-dr**
Updates the `NetworkSwitcher` design and moves it into the `Sidebar` to import IA and reduce layout complexity.

image

### CODE REVIEW

File: components/Shared/NetworkSwitcherGlyph/index.jsx [hunk 1]
```diff
@@ -11,14 +11,15 @@ const NetworkSwitcherButton = styled.button.attrs(() => ({
...
+  paddingRight: 4,
+  paddingTop: 1,
+  paddingBottom: 1,
   fontSize: 2,
   fontWeight: 1,
-  fontFamily: 'RT-Alias-Medium',
+  fontFamily: 'RT-Alias-Grotesk',
   border: 0,
-  borderRadius: 0
+  borderRadius: 4
```
Comment (Schwartz10) [inline components/Shared/NetworkSwitcherGlyph/index.jsx:22]: @as-dr wondering about this border radius - there are no 4th indexed border or border radius properties in our theme - is this pixels? Can we add a theme for this instead for consistency?

**Review by Schwartz10 — CHANGES_REQUESTED** [review]

File: components/Shared/NetworkSwitcherGlyph/index.jsx (hunk 1 above)
Comment (as-dr) [reply components/Shared/NetworkSwitcherGlyph/index.jsx:22]: Very weird. I added it in in another branch previously. Maybe it was overwritten? Will fix.

Comment (as-dr) [reply components/Shared/NetworkSwitcherGlyph/index.jsx:22]: The theme file does have these values, but they are under `radii` --

`  radii: ['0', '1px', '4px', '8px', '16px', '32px', '100px'],`

**Comment by as-dr** [comment]
@Schwartz10 In the Inspector I noticed that the `background` property is returning `true` rather than any color value, which is tied to here:

```
background: ${props => {
    if (props.connected && props.active)
      return props.theme.colors.status.success.background
    if (props.error) return props.theme.colors.status.error.background
    return props.theme.colors.core.transparent
  }};```

Will fix and commit.

File: components/Shared/NetworkSwitcherGlyph/index.jsx (hunk 1 above)
Comment (Schwartz10) [reply components/Shared/NetworkSwitcherGlyph/index.jsx:22]: My bad, didn't know that

**Pull request merged**